extensions/Worker.Extensions.Http/src/FromBodyAttribute.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Functions.Worker.Converters;
+using Microsoft.Azure.Functions.Worker.Extensions.Http.Converters;
+
+namespace Microsoft.Azure.Functions.Worker.Http
+{
+    /// <summary>
+    /// Specifies that a parameter should be bound using the request body.
```

Comment:
Should we also specify this is valid to use for HttpTrigger , in the method summary ?